### PR TITLE
Add postgresql94server test for SLE12

### PR DESCRIPTION
**Pull request by rwmanos**
Appended _sle11 to the postgresql94server.pm test for SLE11.
And created a new test file for SLE12, including testing of basic functionality (createdb, INSERT, SELECT).

### CODE REVIEW

**Comment by rwmanos** [comment]
Working example: http://rwmanosvm2.suse.cz/tests/665#step/postgresql94server/15

File: tests/console/postgresql94.pm [hunk 1]
```diff
@@ -11,17 +11,18 @@
...
 
     select_console 'root-console';
 
     # install the postgresql94 client package
-    assert_script_run "zypper -n in postgresql94", 200;
+    zypper_call "in postgresql94";
 
     # check the postgresql94 client
-    assert_script_run "/usr/bin/psql --help", 200;
+    assert_script_run "/usr/bin/psql --help";
```
Comment (okurz) [inline tests/console/postgresql94.pm]: I am not sure you should reduce the timeout here. The default is 90 seconds. I guess there is a reason why there is "200". Startup could take some time, I guess.

File: tests/console/postgresql94server.pm [hunk 2]
```diff
@@ -11,21 +11,28 @@
...
 
     select_console 'root-console';
 
     # install the postgresql94 server package
-    assert_script_run "zypper -n in postgresql94-server", 200;
+    zypper_call "in postgresql94-server";
 
     # start the postgresql94 service
-    assert_script_run "/etc/init.d/postgresql start", 200;
+    assert_script_run "systemctl start postgresql.service";
```
Comment (okurz) [inline tests/console/postgresql94server.pm]: also here

File: tests/console/postgresql94.pm (hunk 1 above)
Comment (rwmanos) [inline tests/console/postgresql94.pm]: I'm pretty sure this is just the help page, but I'll leave it untouched :+1:

File: tests/console/postgresql94server.pm (hunk 2 above)
Comment (rwmanos) [inline tests/console/postgresql94server.pm]: It worked like this in the 'working example' but it probably makes sense to give it more time :+1:

**Pull request merged**